Source/UserManagement/Web/Controllers/StaffUserController.cs

```diff
@@ -0,0 +1,93 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) 2017 International Federation of Red Cross. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Concepts;
+using doLittle.Domain;
+using Domain;
+using Events;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using doLittle.Events;
+using Read;
+using Infrastructure.AspNet;
+using Read.StaffUsers;
+
+namespace Web
+{
+    [Route("api/staffusers")]
+    public class StaffUserController : BaseController
```

Comment:
This is not how it should be done.  The controller should just delegate to the CommandHandler.